spec/unit/views/pages/form_spec.rb

```diff
@@ -16,7 +16,7 @@
     end
 
     let(:arbre_context) do
-      OpenStruct.new(params: params, helpers: helpers, assigns: {})
+      Struct.new(:params, :helpers, :assigns).new(params, helpers, {})
```

Comment:
This is required to allow specs to pass. Apparently, `ostruct` should be explicitly required after some changes to the dependencies but it is not used on `master` anymore and I've changed it to struct for parity

Reply:
Yes, that's right, thank you for doing this as we don't want to use OpenStruct in general.